wrappers/PhyML/README.md

```diff
@@ -0,0 +1,65 @@
+# **PhyML Galaxy Wrapper** <br>
+This repository contains a Galaxy tool wrapper for the PhyML program, which estimates the maximum likelihood phylogenies from alignments of nucleotide or amino acid sequences. <br>
+PhyML has been developed within the PhyML Development Team at ATGC Montpellier Bioinformatics platform. (*http://www.atgc-montpellier.fr/phyml/*)
+
+The reference for PhyML is: <br>
+	<pre>*New Alogirthms and Methods to Estimate Maximum-Likelihood Phylogenies: Assessing the Performance of PhyML* 
+Guindon S., Dufayard J.F., Lefort V., Anisimova M., Hordijk W., & Gascuel O. (2010). 3.0. 
+System Biology, 59(3):307-21. </pre>
+
+The PhyML Tool Suite repository: <br>
+<pre>* Provides PhyML wrapper for the PhyML tool 
+* Downloads and installs PhyML on the Linux operating system  </pre>
+
+Requirements: <br>
+<pre>* Linux 
+* Virtual Machine 
```

Comment:
This is very general, and probably not necessary. We run Galaxy with vagrant, but it is not a requirement.